CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Version 0.5.0-dev  (changes since 0.5.0 go here)
 
 ### Features
+* DissimilarityMatrix now has a new constructor method called from_iterable.
```

Comment:
Can you add the issue number that this fixes? Would also be nice to backtick DissimilarityMatrix and from_iterable for readability.